src/Adapter/Tornado/SynchronousEventLoop.php

```diff
@@ -156,11 +156,12 @@ public function deferred(): Deferred
         $deferred = new class() implements Deferred {
             /** @var SynchronousEventLoop */
             public $eventLoop;
-            private $promise;
+            /** @var ?Promise */
+            private $promise = null;
 
             public function getPromise(): Promise
             {
-                if (!$this->promise) {
+                if ($this->promise === null) {
```

Comment:
@b-viguier The other "real" change to the code, it seems safe :sweat_smile:

Reply:
👌